agent.go

```diff
@@ -224,3 +224,25 @@ func defragment(gcfg globalConfig, ep string) error {
 	_, err = c.Defragment(ctx, ep)
 	return err
 }
+
+func transferLeadership(gcfg globalConfig, leaderEp string, transfereeID uint64) error {
+	cfgSpec := clientConfigWithoutEndpoints(gcfg)
+	cfgSpec.Endpoints = []string{leaderEp}
+	c, err := createClient(cfgSpec)
+	if err != nil {
+		return fmt.Errorf("failed to create client for leader endpoint %s: %w", leaderEp, err)
+	}
+	defer c.Close()
+
+	ctx, cancel := commandCtx(gcfg.commandTimeout)
+	defer cancel()
+
+	fmt.Printf("Requesting leader at %s to transfer leadership to member ID %d...\n", leaderEp, transfereeID)
```

Comment:
Suggest not to print log in this utility function, please print log in main.go only if needed
```suggestion
```